typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -70,4 +70,5 @@ export enum WarpRouteIds {
   SolanaSonicsvmUSDStar = 'USDStar/solanamainnet-sonicsvm',
   SolanaSonicsvmSSOL = 'sSOL/solanamainnet-sonicsvm',
   ArbitrumEthereumSolanaTreasureSMOL = 'SMOL/arbitrum-ethereum-solanamainnet-treasure',
+  BaseSolanaCDX = 'CDX/base-solanamainnet',
```

Comment:
are we not including this in check-warp-deploy? since we didn't add an entry in `warpConfigGetterMap`